### PR TITLE
notes from first developer call

### PR DESCRIPTION
**Pull request by romanlutz**
Adding the writeup - please comment if you want anything to be added/changed!

If there are no comments I'll complete the PR by Friday 5/22

### CODE REVIEW

File: docs/community/developer_call_notes/2020_05_07.rst [hunk 1]
```diff
@@ -1,2 +1,101 @@
...
+- `Hanna Wallach <https://www.microsoft.com/en-us/research/people/wallach/>`_
+
+Organizational updates
+^^^^^^^^^^^^^^^^^^^^^^
+
+#. Microsoft & Fairlearn:
+
+    - goal is to be an open source, community driven project
+    - sociotechnical framing is central to the project, careful and precise
+      wording matter
```
Comment (MiroDudik) [inline docs/community/developer_call_notes/2020_05_07.rst:29]: A couple small suggestions:
* maybe split this off the first two sub bullets separately under `#. Fairlearn project`
* before the last two sub-bullets add one more explanatory sub-bullet "currently many core members are from Microsoft, but the project is an independent open source project" (a bit repetitive, but doesn't hurt)
* it seems that the sub-bullets jumped an extra level (just me being nitpicky)

Comment (romanlutz) [reply docs/community/developer_call_notes/2020_05_07.rst:29]: which ones jumped? 
Adding all the changes you suggested except for the last one which I don't understand yet.

Comment (romanlutz) [reply docs/community/developer_call_notes/2020_05_07.rst:29]: should look like this now 
image

**Comment by romanlutz** [comment]
> This is pretty cool, and I really like the attention you've put into this. Just a note that I rather have the notes even if not perfect, in case it becomes too much work as we go on. But I really like this one.

Absolutely! I don't mind even making adjustments later on if people want something reworded or fix typos or what not. 
In fact, I'm unhappy with how long it took me to get this PR out. That was due to the webpage/documentation generation taking a couple days longer than I thought, so at least it won't happen next time.

**Comment by adrinjalali** [comment]
one thing I tend to do in meetings is to take the minutes as the meeting goes, on something shared (like hackmd.io) where people can see and edit together, and by the time the meeting is over, the minutes are ready to be published.

**Comment by romanlutz** [comment]
> one thing I tend to do in meetings is to take the minutes as the meeting goes, on something shared (like hackmd.io) where people can see and edit together, and by the time the meeting is over, the minutes are ready to be published.

I love that. Let's see what makes sense! I don't want people to have to sign up for too many new tools, but this seems promising

**Pull request merged**